lms/templates/video.html

```diff
@@ -33,12 +32,10 @@ <h4 class="video-error hidden">${_('No playable video sources found.')}</h4>
                   <div class="secondary-controls"></div>
               </div>
           </section>
-          <a class="nav-skip sr" id="before-transcript_${id}" href="#after-transcript_${id}">${_('Skip to end of transcript.')}</a>
```

Comment:
I've added the links to skip back and forth between the start and end of the transcript.